move.h

```diff
@@ -0,0 +1,12 @@
+#ifndef MOVE_H
+#define MOVE_H
+
+#include "element.h"
+
+struct Move {
+    Element gem;
+    Element source;
+    Element destination;
+};
+
+#endif
```

Comment:
Add missing newline at end of file.
```suggestion
#endif

```